### PR TITLE
Add LIBRARY_TASK_MAPPING_EXCLUDING_TRANSFORMERS

### PR DESCRIPTION
**Pull request by xianbaoqian**
Context:  https://huggingface.slack.com/archives/C02ERUY9TKK/p1669743642830359

### CODE REVIEW

File: js/src/lib/interfaces/Types.ts [hunk 1]
```diff
@@ -749,3 +749,97 @@ export interface TransformersInfo {
 	 */
 	processor?: string;
 }
+
+
+/**
+ * Mapping from library name (excluding Transformers) to its supported tasks. 
+ * Inference API should be disabled for all other library, task pairs beyond this mapping.
```
Comment (osanseviero) [inline js/src/lib/interfaces/Types.ts:756]:
```suggestion
 * Inference API should be disabled for all other (library, task) pairs beyond this mapping.
```

File: js/src/lib/interfaces/Types.ts [hunk 2]
```diff
@@ -749,3 +749,97 @@ export interface TransformersInfo {
...
+ * As an exception, Transformers supports all inference tasks.
```
Comment (osanseviero) [inline js/src/lib/interfaces/Types.ts:757]:
```suggestion
 * As an exception, we assume Transformers supports all inference tasks.
```

**Review by osanseviero — APPROVED** [review]
Very nice!

**Comment by xianbaoqian** [comment]
Thanks Omar for the detailed review!

**Pull request merged**